src/p2a_analyses_read_processed_data.R

```diff
@@ -27,7 +27,7 @@ data$yday <- as.numeric(data$yday)
 data$origin <- as.character(data$origin)
 
 # Gather Swedish data with all the same columns as df "data" created above
-source('sidescript_readSwedes.R')
+source("src/sidescript_readSwedes.R")
```

Comment:
Please make sure to take into account the suggestion of @peterdesmet regarding the usage of the `here` package, see https://github.com/inbo/wmh-analysis/issues/1

Reply:
same refers for all other changes to the relative paths

Reply:
I made 2 further changes to the code: 
1. The `here` package is used for the relative paths now
2. I checked whether all scripts run without any errors and made some small corrections for that matter.